content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -85,14 +78,56 @@ datasource db {
 }
 ```
 
-#### Step 5: Generate the client
+##### Vercel Edge
+
+If you want to use Vercel Edge, then
+**!!! (inject the environment variables) - explanation to go here!!!**
+
+##### Cloudflare Workers
+
+If you want to use CloudFlare Workers then
+**!!!! (use global variables) - explanation to go here!!!**
 
-To generate the client using the Data Proxy, set the environment variable `PRISMA_CLIENT_ENGINE_TYPE` to `dataproxy` using the following command:
+#### Step 4: Generate the client
+
+To generate the client with the Data Proxy, call `prisma generate` with the option `--data=proxy`:
 
 ```terminal
-PRISMA_CLIENT_ENGINE_TYPE='dataproxy' npx prisma generate
+prisma generate --data proxy
 ```
 
+The Data Proxy does not need local engine files. To keep Prisma Client lightweight, when you use this command the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine) in the generated Prisma Client.
```

Comment:
```suggestion
The Data Proxy does not need local engine files. To keep Prisma Client lightweight, use the above command because the generation process does not include the [query engine](/concepts/components/prisma-engines/query-engine) in the generated Prisma Client.
```

Reply:
You don't run the command to keep the client lightweight. You run the command to use the Data Proxy. The lightweight-ness is a positive side effect of using the Data Proxy. However, I can see the confusion, so I'll clarify this paragraph.

Reply:
Thanks. I've reworked the paragraph to remove this confusion.